build.gradle

```diff
@@ -54,7 +54,7 @@ ext {
 		cucumber_junit_version             : "7.11.2"
 	]
 	// Code Coverage (Lines/Branches) cannot be below this value on a per sub project basis
-	minUnitTestLineCoverage = 0.9
```

Comment:
Restore this back to 90% and add the new lambda to the aggregate report.

https://github.com/govuk-one-login/ipv-cri-uk-passport-api/blob/cd1b4a64df1c7bfec5564ff28ae3f3121bfcfd3a/build.gradle#L81